branca/element.py

```diff
@@ -43,11 +43,11 @@ def __init__(self, template=None, template_name=None):
         self._parent = None
         self._template = Template(template) if template is not None\
             else ENV.get_template(template_name) if template_name is not None\
-            else Template(u"""
-        {% for name, element in this._children.items() %}
-            {{element.render(**kwargs)}}
-        {% endfor %}
-        """)
+            else Template(
+                "{% for name, element in this._children.items() %}\n"
+                "    {{element.render(**kwargs)}}"
+                "{% endfor %}"
+                )
```

Comment:
Yes, it's the purpose. See http://nbviewer.jupyter.org/github/bibmartin/branca/blob/cleanup/examples/Elements.ipynb